README.md

```diff
@@ -196,5 +196,25 @@ save them as the Volapük (vo) translation. Volapük is used because it's a cons
 in react-intl and an estimated 20 speakers worldwide. To switch to the test translations, add "Volapük" to the top of
 your browser's preferred language list. In Firefox, this can be configured in the Language section of about:preferences.
 
+### Making A New Release Of welder-web
+
+When the project is ready for a new release, do the following:
+
+ * Tag the release with `make NEWTAG=X.Y.Z tag`, bumping .Z to the next version unless there are major changes.
+ * Edit the commit list to reflect the changes that will be visible to users (it shows up on the GitHub Releases page)
+ * Sign the tag with your GPG key
+
+(and editor should open automatically, and gpg is required to be setup in order to sign the tag).
+
+This will trigger [cockpituous](https://github.com/cockpit-project/cockpituous/tree/master/release) to build a new release of welder-web.
+
+#### Releasing cockpit-composer
+
+Do this after tagging a new release of `welder-web` as described above. It uses the same source.
+
+ * Pull the latest translations by running `make po-pull`
```

Comment:
`make cockpit-composer-srpm` already does the po-pull (via the dist-gzip dependency), so this step isn't necessary. I don't mind if we want it to be an explicit step, it just isn't at the moment.